docs/UserGuide.md

```diff
@@ -97,7 +121,7 @@ Examples:
 Edits an existing expense in the list. You need to specify the index of the expense from the list and only the syntax of the fields you want to change together
 with the value you want to change to. You can specify one or more fields. Run `list` before executing the command to identify the index of the expense.
 
-Format: `edit INDEX [d/DESCIPTION] [a/AMOUNT] [c/EXPENSE_CATEGORY] [t/EXPENSE_DATE]`
+Format: `edit INDEX (d/DESCRIPTION) (a/AMOUNT) (c/EXPENSE_CATEGORY) (t/EXPENSE_DATE)`
```

Comment:
The square brackets mean the argument in optional, again not something we are supposed to change i think.

Reply:
Andy came up with round brackets, which means at least one of the inputs in round brackets must be present. I think it's more relevant in this case since we need at least 1 field to be edited.